appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1201,9 +1201,15 @@ definitions:
           - $ref: '#/definitions/SelfLink'
           - $ref: '#/definitions/EditLink'
           - $ref: '#/definitions/ActivityCategoryLink'
+      creationTime:
+        type: string
+        format: dateTime
+        description: The creation time of this goal
+        example: 2016-04-26T20:08:55.365+02:00
```

Comment:
How should I do that? Is it a Jackson setting? The type is different: the other places serialize `Date`, this place serializes `ZonedDateTime`...